lib/blueprintLocations.js

```diff
@@ -27,14 +27,16 @@ module.exports = function($) {
             let itemdropchance = $(tr.children[1]).text()
 
             // enemyBlueprintDropChance is legacy, should be removed ASAP...
-            item.enemies.push({
+            let enemy = {
                 _id: hash(text),
                 enemyName: text,
                 enemyItemDropChance: Number(itemdropchance.slice(0, itemdropchance.length - 1)),
                 enemyBlueprintDropChance: Number(itemdropchance.slice(0, itemdropchance.length - 1)),
                 rarity: chance.rarity,
                 chance: Number(chance.chance)
-            })
+            }
+            
+            addUniqueHashedObject(item.enemies, enemy, ["enemyName", "enemyItemDropChance", "chance"])
```

Comment:
Why change how we hash....again?

Seems like something that should be in the PR notes instead of "a bunch of changes, everything 'should' be in order".

Reply:
It doesn't. While working on my other project I noticed that these locations still have issues, therefor not _completely_ resolving the original intention of the previous PR. This is just implementing the previous PR in all locations this is useful.

Sorry about leaving that out. It seemed to me it's counter-productive to list all changes again in the PR while I tried to explain everything I did in the commits already. Well, apart from the explanation above. I have added it to the PR. Missed the whole reason for this PR, derp.